### PR TITLE
Update GraphQL schema for labeling

### LINKED ISSUE #122: Update GraphQL schema for Labels
**AC**
- Modify Label scalar  (key: value)
- Update schema for label filtering. Implementation will be done in a separate task.
- Prepare GraphQL types for LabelDefinition
- CRUD for LabelDefinition (queries and mutations)
- Rename `addApplicationLabel` to `setApplicationLabel` (for runtimes too)

These changes should be done according to proposal from https://github.com/kyma-incubator/compass/pull/149. 

Remember about integration tests.

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Update GraphQL schema for labeling

**Related issue(s)**
Resolves #122

### CODE REVIEW

File: components/director/pkg/graphql/schema.graphql [hunk 1]
```diff
@@ -444,16 +458,10 @@ input BasicCredentialDataInput {
...
 }
 
-# HealthCheck Input
-
-enum FilterOperator {
-    ALL, ANY
-}
-
 input LabelFilter {
     label: String!
```
[the remark this answers is not in the document]
Comment (pkosiec) [reply components/director/pkg/graphql/schema.graphql]: yup, `key` sounds a little bit better

File: components/director/pkg/graphql/schema.graphql [hunk 2]
```diff
@@ -444,16 +458,10 @@ input BasicCredentialDataInput {
...
-    values: [String!]!
-    operator: FilterOperator = ALL
+    """Optional SQL/JSON Path expression. If not provided, returns every object with given label regardless of its value."""
```
[the remark this answers is not in the document]
Comment (pkosiec) [reply components/director/pkg/graphql/schema.graphql]: Probably we will need to point to our document (which doesn't exist right now), where we'll write which operations are supported.

[the remark this answers is not in the document]
Comment (aszecowka) [reply components/director/pkg/graphql/schema.graphql]: maybe create an issue for that and add a link to it

[the remark this answers is not in the document]
Comment (pkosiec) [reply components/director/pkg/graphql/schema.graphql]: Yup, I mentioned this issue in the TODO comment: https://github.com/kyma-incubator/compass/issues/163

**Pull request merged**